Workbooks/Workloads/Onboard MySQL Insights/Onboard MySQL Insights.workbook

```diff
@@ -156,7 +156,7 @@
                 {
                   "type": 1,
                   "content": {
-                    "json": "MySQL is one of the world’s most popular open source database. With its proven performance, reliability, and ease-of-use, MySQL has become the leading database choice for web-based applications, used by high profile web properties including Facebook, Twitter and YouTube.\n\nWith Azure Monitor for MySQL, you can manage your MySQL while also gaining visibility into your underlying host – Windows or Linux. Make configuration a breeze with templates, automated alerts and customizable actions. Spend less time fighting fires by accurately diagnosing real-time and historical load balancer performance issues, and focus on more proactive measures scale outs.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"
+                    "json": "MySQL is the world's most popular Open Source SQL database management system. MySQL is highly scalable and can be operated from single desktops to clusters of machines. MySQL also offers hallmark reliability and availability features from master/slave replication configurations and specialized failover clusters. \n\nWith Azure Monitor for MySQL you can manage your MySQL servers with ease. With insights on performance, networking, and InnoDB, Azure Monitor for MySQL summarizes all the important metrics to keep your MySQL server up and running. Make configuration a breeze with templates, automated alerts and customizable actions.\n<br />\n<br />\n[Watch Video]()<br />\n[Learn More]()<br />\n[Documentation]()<br />"

```

Comment:
We should take out the links at the bottom for the private preview or until we have actual working links.